packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -47,6 +48,7 @@ export function createApp() {
   const app = new OrchestratorApiBuilder(new TupaiaDatabase())
     .useSessionModel(AdminPanelSessionModel)
     .verifyLogin(hasTupaiaAdminPanelAccess)
+    .get('/v1/hierarchy/:hierarchyName/:entityCode', handleWith(EntitiesRoute))
```

Comment:
We also need the root project entity code here to get all the entities of a hierarchy. I think the `/projects` endpoint in meditrak-server can supply that.